includes/widgets/icon-list.php

```diff
@@ -424,12 +447,29 @@ protected function register_controls() {
 			[
 				'label' => esc_html__( 'Size', 'elementor' ),
 				'type' => Controls_Manager::SLIDER,
+				'size_units' => [ 'px', '%', 'em', 'rem' ],
 				'default' => [
+					'unit' => 'px',
 					'size' => 14,
 				],
 				'range' => [
```

Comment:
Change to:
```
'range' => [
		'px' => [
			'min' => 6,
			],
		'%' => [
			'min' => 5,
			],
		'em' => [
			'min' => 1,
			],
		'rem' => [
			'min' => 1,
			],
	],
```

The reason why we added the min value to avoid situation that the icon is hidden when the user choose `0` value

Reply:
Done.